README.md

```diff
@@ -298,3 +298,37 @@ MirrorMaker2 should size `tasks.max` so that each MirrorSourceConnector task has
 * **Why it matters:** The number of records per task shows the aggregate throughput that Kafka Connect is processing. Each MSC task should be balanced with a similar number of overall records, and any MSC task with many more or fewer records indicates an unbalanced MirrorMaker2 replication state. This likely indicates a hot partition, or an incorrect setting for `tasks.max`.
 
 
+### I am getting `InvalidRecordException` exception with timestamp out of range error
+
+* **What is the error:** This shows as an exception which causes the tasks fail and replication stops. It could happen right after you start the source connector, or when the connector has stopped for a long time and you restart it again. This error usually shows in logs as follows:
+
+```sh
```

Comment:
Remove `sh` - this is plaintext so we don't need a language for syntax highlighting

Reply:
updated